### PR TITLE
feat: MT3-737 change label on approve button and date of submission

### PR DESCRIPTION
**Pull request by LBHABEBIC**
Unable to enter manager review button doesn't need an option to decline the process as the HO has exhausted all possibilities by attempting 4 visits.

image

### CODE REVIEW

File: components/managerApprovedButton.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+  });
+
+  return (
+    <Button
+      preventDoubleClick
+      disabled={disabled}
+      className={classNames({
+        "lbh-button--warning govuk-button--warning":
+          status === ProcessStage.Approved,
+      })}
```
Comment (mrmartineau) [inline components/managerApprovedButton.tsx]:
```suggestion
```

I'm not sure this block is needed. Should this button be red / warning if the user is approving the process?

**Review by mrmartineau — CHANGES_REQUESTED** [review]
see my comment, not sure if a change is needed..

File: components/managerApprovedButton.tsx (hunk 1 above)
Comment (LBHABEBIC) [reply components/managerApprovedButton.tsx]: good point.i'll fine the class for a green button.

**Review by mrmartineau — APPROVED** [review]
💯

**Pull request merged**